docs/extend/extend-apiml/api-mediation-onboard-a-sprint-boot-rest-api-service.md

```diff
@@ -245,7 +245,8 @@ The first step to onboard a REST API with the Zowe ecosystem is to add enabler a
       spring:
           application:
               name: ${mfaas.discovery.serviceId}      
-
+    ```
+    **Note:** Add this configuration also to the `application.yml` used for testing.
```

Comment:
Can you explain in this note why it is important to add this configuration to application.yml used for testing?